### PR TITLE
deniseddi/C05-make01

### PR DESCRIPTION
**Pull request by deniseddi**
# Description

Exercise (e.g. C01-E01):  C05-make01

Student Name: Denise Horstmann

Completed the first exercise on "make". 
It was a little challenging and took me some time to write the correct code and syntax,  but once it was all working fine and generating outputs, looking back at the Dockerfile and Makefile, they seem quite simple actually. Good start. Great automation tool!  

## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/05class/exercises/c05-make01/deniseddi/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM httpd:2.4.41-alpine
+COPY index.html /usr/local/apache2/htdocs/
+# EXPOSE 80
```
Comment (denstorti) [inline classes/05class/exercises/c05-make01/deniseddi/Dockerfile:3]: Hi Denise, why is this line commented out? You can leave it, it works as documentation for your docker file

>The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.

File: classes/05class/exercises/c05-make01/deniseddi/output.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+1dfb4a5d0ecd: Pushed
+3313c0c04ade: Pushed
+310889822143: Pushed
+beee9f30bc1f: Pushed
+4e24982: digest: sha256:564a70c9997d173727d55a72d5ed17c0a9675706224c59ea94c0998377a7ba7c size: 1569
+
+#Output of command "make run"
+\$ make run
+docker run -p 80 -d portoddi/make01:4e24982
+203202a88413f720660dc4034717660a5919386b980f5ae307bc88e061c6887b
```
Comment (denstorti) [inline classes/05class/exercises/c05-make01/deniseddi/output.md:39]: Can you include the logs without detaching the container so I can actually see the output?

Comment (deniseddi) [reply classes/05class/exercises/c05-make01/deniseddi/output.md:39]: Hi Denis!
I am reviewing this exercise.
I left the "Expose 80" uncommented :) 

Now, without the detach command `denisegomes$ make run
docker run -p 80 portoddi/make01:dd2d4f0` I get this output: (Is this what you expected?) 

docker run -p 80 portoddi/make01:dd2d4f0
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
[Mon May 11 06:20:13.006856 2020] [mpm_event:notice] [pid 1:tid 139903113719112] AH00489: Apache/2.4.41 (Unix) configured -- resuming normal operations
[Mon May 11 06:20:13.009952 2020] [core:notice] [pid 1:tid 139903113719112] AH00094: Command line: 'httpd -D FOREGROUND'
[Mon May 11 06:20:26.306513 2020] [mpm_event:notice] [pid 1:tid 139903113719112] AH00492: caught SIGWINCH, shutting down gracefully

Comment (starkmatt) [reply classes/05class/exercises/c05-make01/deniseddi/output.md]: That sounds like what Denis asked for 😄 !

Comment (denstorti) [reply classes/05class/exercises/c05-make01/deniseddi/output.md]: Glad to see you commenting here @starkmatt !
Feel free to jump in on PRs of the new students too, comment and approve them 🤝 

Thanks @deniseddi

**Pull request merged**